Formula/p/parallel-hashmap.rb

```diff
@@ -1,9 +1,11 @@
 class ParallelHashmap < Formula
   desc "Family of header-only, fast, memory-friendly C++ hashmap and btree containers"
   homepage "https://greg7mdp.github.io/parallel-hashmap/"
-  url "https://github.com/greg7mdp/parallel-hashmap/archive/refs/tags/1.37.tar.gz"
-  sha256 "2ac652be0552fcb53a1163c08c1f28f29f0756594fcc587eebb4d8b363153709"
+  url "https://github.com/greg7mdp/parallel-hashmap/archive/refs/tags/v1.3.12.tar.gz"
+  sha256 "0cc203144321924cfbfcc401f42d8204c0dd24e2760c7a1c091baa16d9777c08"
   license "Apache-2.0"
+  # upsteam versioning scheme changed between v1.37 and v1.3.8
+  version_scheme 1
```

Comment:
I would think we should use downgrade instead cc @samford

Reply:
Even though this *is* an actual upstream versioning scheme change? What's the purpose of `version_scheme` if it's not that? :)

Reply:
I think you are right, yeah, I was mostly thinking to avoid introducing version_scheme at the best efforts :)

Reply:
IMHO, it would have been best if upstream had made the change in a backwards-compatible way (i.e. following up with v1.38.0 after v1.37), but I'm really glad to see that Homebrew has good way of dealing with this!

Reply:
Yep, we try to avoid adding `#version_scheme` when possible but it's necessary/appropriate in this scenario.